workflow/scripts/generate-overview-table.py

```diff
@@ -170,4 +171,11 @@ def register_contig_lengths(assemblies, name):
     )
     data.loc[sample, "other variants"] = " ".join(sorted(set(other_variants)))
 
+fmt_dict = {
+    int : ["# raw reads", "# trimmed reads", "# used reads", "initial contig (bp)", "final contig (bp)"],
+    str : ["pangolin strain (#SNPs)", "variants of interest", "other variants"]
+}
+for dtype, columns in fmt_dict.items(): 
+    data[columns] = data[columns].astype(dtype)
```

Comment:
Are you sure this is needed? If it is needed, it means to me that we insert the values in the wrong datatypes.

Reply:
Ahh correct, I didn't think enough, when I did that. Removed the `str` casting in 55e5c15.